keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -320,11 +608,16 @@ maxUnavailable to a number greater than 1,  but the invariants and the logic wil
 maxUnavailable pods with the same identity and never more than maxUnavailable being deleted.
 
 ###### What specific metrics should inform a rollback?
-TODO when we reach Beta
+
+When feature enabled but rolling update in a unexpected phenomenon like the update pods at a time is not equal to the
+`maxUnavailable` value or rolling update in a unexpected order.
+
+Or we can refer to the `rolling-update-duration` metric for observation, if it didn't decrease when setting the `maxUnavailable`
+great than 1 or the duration increased abnormally, then we should rollback.
 
```

Comment:
Are there other factors that may increase the rolling update duration? For example, if maxUnavailable is set to 2 but for some reason the rolling upgrade takes longer than when maxUnavailable was 1.

Maybe we should just focus on metrics such as "number of unavailable pods" and compare it with maxUnavailable to inform a rollback.

Reply:
> Are there other factors that may increase the rolling update duration? For example, if maxUnavailable is set to 2 but for some reason the rolling upgrade takes longer than when maxUnavailable was 1.

Exactly we do, let me add an explanation like `We may have some other factors impacting this, but this should apply for most of the time.

> Maybe we should just focus on metrics such as "number of unavailable pods" and compare it with maxUnavailable to inform a rollback.

If you mean adding a new metric, then the metric value is equal to `maxUnavailable` for most of the time, then it has not significant values.